lib/i18n.rb

```diff
@@ -204,7 +207,9 @@ def translate!(key, options = EMPTY_HASH)
     alias :t! :translate!
 
     # Returns true if a translation exists for a given key, otherwise returns false.
-    def exists?(key, locale = config.locale)
+    def exists?(key, _locale = nil, locale: _locale)
```

Comment:
I also added this. While crafting the error message I figured that `exists?` was inconsistent with the others, as it would take a locale as second param instead of a named one.

Having all methods accepting it as named param allows for a more insightful error message.